docs/contributing/style-guide.md

```diff
@@ -0,0 +1,420 @@
+
+Use active voice instead of passive voice, because active voice is more concise and easier to understand.
+
+When using voice, the subject of the sentence is performing the action. In passive voice, the subject is being acted upon. A famous example of passive voice is the phrase "mistakes were made."
+
+```
+# Not recommended (passive)
+When an alarm is triggered by a metric, a notification is sent by Netdata...
+
+# Recommended (active)
+When a metric triggers an alarm, Netdata sends a notification...
+```
+
+### Standard American spelling
+
+While the Netdata team is mostly *not* American, we still aspire to use American spelling whenever possible, as it is more commonly used within the monitoring industry.
+
+### Clause order
+
+If you want to instruct your reader to take some action in a particular circumstance, such as optional steps, the beginning of the sentence should indicate that circumstance.
+
+```
+# Not recommended
+Read the reference guide if you'd like to learn more about custom dashboards.
+
+# Recommended
+If you'd like to learn more about custom dashboards, read the reference guide.
+```
+
+By placing the circumstance at the beginning of the sentence, those who don't want to follow can stop reading and move on. Those who *do* want to read it are less likely to skip over the sentence.
+
+### Oxford comma
+
+The Oxford comma is the comma used after the second-to-last item in a list of three or more items. It appears just before "and" or "or."
+
+```
+# Not recommended
+Netdata can monitor RAM, disk I/O, MySQL queries per second and lm-sensors.
+
+# Recommended
+Netdata can monitor RAM, disk I/O, MySQL queries per second, and lm-sensors.
+```
+
+
+## Markdown syntax
+
+The Netdata documentation uses the Markdown syntax for styling and formatting. If you're not familiar with how it works, please read the [Markdown introduction post](https://daringfireball.net/projects/markdown/) by its creator, followed by [Mastering Markdown](https://guides.github.com/features/mastering-markdown/) guide from GitHub.
+
+We also leverage the power of the [Material theme for MkDocs](https://squidfunk.github.io/mkdocs-material/), which features several [extensions](https://squidfunk.github.io/mkdocs-material/extensions/admonition/), such as the ability to create notes, warnings, and collapsible blocks.
+
+You can follow the syntax specified in the above resources for the majority of documents, but the following sections specify a few particular use cases.
+
+### References to UI elements
+
+If you need to instruct your reader to click a user interface (UI) element inside of a Netdata interface, you should reference the label text of the link/button with Markdown's (`**bold text**`) tag.
+
+```markdown
+Click on the **Sign in** button.
+```
+
+!!! note
+    
+    Avoid using directional language to orient readers, because not every reader can use instructions like "look at the top-left corner" to find their way around an interface.
+
```

Comment:
How do we reference the menu formerly known as `my-netdata`, currently mentioned as `nodes` when it has a dynamic name?
How do we reference the right navbar?
These are good examples to put here.